spec/request/service_credential_bindings_spec.rb

```diff
@@ -646,6 +657,38 @@ def check_filtered_bindings(*bindings)
       end
     end
 
+    context "last binding operation is in 'delete in progress' state" do
+      before do
+        app_binding.save_with_attributes_and_new_operation({}, { type: 'delete', state: 'in progress' })
+      end
+
+      it 'returns an error' do
+        api_call.call(admin_headers)
+        expect(last_response).to have_status_code(404)
+        expect(parsed_response['errors']).to include(include({
+                                                               'detail' => 'Deletion of service binding in progress',
```

Comment:
The formatting (indentation) looks a little bit weird. I would prefer if it was consistent with above.

Reply:
Done :)